test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1343,6 +1350,13 @@ def test_softshrink(self):
         x = torch.tensor(-0.5).to(dtype=torch.float32)
         self.run_test(model, x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_softshrink_dtype(self):
+        model = torch.nn.Softshrink()
+
+        x = torch.rand(3, 3).to(dtype=torch.float64)
+        self.run_test(model, x)
```

Comment:
I would inline the model. Could help create a deterministic input that is sure to have values falling under all <-λ, ==-λ, -λ...λ, ==+λ and >+λ regions to test all cases?
```suggestion
        ...
        self.run_test(torch.nn.Softshrink(), x)
```